Realtime/src/commonMain/kotlin/io/github/jan/supabase/realtime/event/RealtimeEvent.kt

```diff
@@ -0,0 +1,47 @@
+package io.github.jan.supabase.realtime.event
+
+import io.github.jan.supabase.realtime.RealtimeChannel
+import io.github.jan.supabase.realtime.RealtimeMessage
+
+/**
+ * Interface for handling realtime events.
+ */
+internal sealed interface RealtimeEvent {
```

Comment:
We could possibly rename this and subclasses to `RealtimeEventHandler`

Reply:
I'm fine either way since it's an internal symbol 👍